### ISSUE TITLE
A11y_AzureToolsStorageExplorer_UpdateAccessTier_ScreenReader: VoiceOver not announces "Learn More" link in descriptive.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/AzureStorageExplorer#1966]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #MAS2.4.4;#VoiceOver; #Mac; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click 
   on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create
   Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using "Upload Files"
   control.
6. Right click on blob and select "Change Access Tier" control and hit enter.
7. "Update Access Tier" popup gets opened.
8. Start VoiceOver.
9. Navigate using Tab key to "Learn More" link, verify VoiceOver not announces "Learn More" in descriptive way.

**​Actual:​**
While navigating using Tab key to "Learn More" link, VoiceOver not announces "Learn More" link in descriptive way. Instead it announces other content present on the screen with "Learn More" link.

**Expected:**
While navigating using Tab key to "Learn More" link, VoiceOver should announce "Learn More" link in more descriptive way.

**User Impact:**
Screen reader users face difficulty to know the purpose of the link if screen reader not announces link in more descriptive way.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

**Attachment for Reference:**
[1966_A11y_AzureToolsStorageExplorer_UpdateAccessTier_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669479/1966_A11y_AzureToolsStorageExplorer_UpdateAccessTier_ScreenReader.pptx)

**Comment by craxal** [comment]
Same root cause of #1920.

**Comment by craxal** [comment]
@vinodbk197 Can explain what you mean by "not descriptive"? VoiceOver sounds like it reads the "Learn More..." link just fine, albeit it says "Update Access Tier and 2 more items" first, which I think is OK.

**Comment by Suraj1200499** [comment]
@Craxal Actually Screen reader announces Content present above/below the "Learn More" link but not announces learn more link in descriptive.

Voice Over should announce the information that is going to be available after hitting on the "Learn More" link in brief manner, when voice over focus is "Learn More" link.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0 in MAC Version: Catalina 10.15.3, this issue is no longer reproduced. Hence Closing the bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
